public/styles.css

```diff
@@ -11,7 +11,7 @@
   color: rgb(247, 57, 82);
   vertical-align: bottom;
   font-weight: bold;
-  font-size: 20px;
+  font-size: 1.5rem;
```

Comment:
Is using `rem` less reliable than `em`?

The root element will ultimately be shared with everyone else.

Reply:
Great comment! I wanted to be able to scale across the entire page so I used rem opposed to the em's dependency on parents' sizes. I do see your point about potential impact when our modules come together. I'm very interested to see what will happen. It will be a quick adjustment if the conversion is necessary.